requirements/requirements.in

```diff
@@ -114,3 +114,4 @@ CommcareTranslationChecker==0.9.3.5
 WeasyPrint==0.42.3
 architect==0.5.6
 prometheus-client==0.7.1
+git+git://github.com/dimagi/turn-python.git
```

Comment:
How hard would it be to do versioned releases on PyPi? Would be good to know exactly which version of the code is running in production rather than using the latest form github.

Reply:
We are planning on versioning the release going forwards